doc/rls_mode/linting.md

```diff
@@ -0,0 +1,15 @@
+# RLS Mode Linting Page
+
+`RLS` checks the project and shows diagnostics while you are typing.
+
+You can see diagnostics in the Problems Panel.

```

Comment:
Simply, "Panel" is capitalised whereas in previous mentions you said "Problems panel".